department-of-veterans-affairs/va.gov-cms
# VACMS-10089: Checkbox to add event to National Outreach Calendar

## Description

Relates to #10089 

## Testing done
Functional in CMS, cypress

## Screenshots
![Screenshot 2023-10-05 at 2 45 48 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/22764938/5ba1c04a-260d-4727-9efd-f2880fc3f6e3)
![Screenshot 2023-10-05 at 2 45 29 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/22764938/66b02a54-3743-4828-8e82-babfb0a7f3ff)


## QA steps

As a user with content_admin role:
- Go to /add/node/event
- [ ] Check that there is a checkbox under "Where should this event be listed" that says "Publish to National Outreach Calendar"
- Check it, fill out essential form elements and save
- [ ] Check that the "Outreach events" link shows on the node view page

As a user with the roles office_content_creator, content_publisher
- Go to /add/node/event
- [ ] Check that there is a checkbox under "Where should this event be listed" that says, "Event will automatically be published to National Outreach Calendar"

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
